src/backend/tools.ts

```diff
@@ -393,15 +393,12 @@ export const DXVK = {
 
 export const Winetricks = {
   download: async () => {
-    if (!isLinux) {
+    if (isWindows) {
       return
     }
 
-    const linuxUrl =
+    const url =
```

Comment:
Also the standard winetricks works fine on mac, no need for a different one than linux.